hq_superset/views.py

```diff
@@ -230,17 +275,24 @@ def refresh_hq_datasource(domain, datasource_id, display_name):
 
 
 @contextmanager
-def get_csv_file(provider, oauth_token, domain, datasource_id):
+def get_datasource_file(path):
```

Comment:
I smell a DataSourceHelper/DataSourceService brewing...In the meantime to satisfy the hunger, I'd suggest this function to live in the `utils.py` file rather to keep the views clean?

Reply:
>  I'd suggest that this function should live in the utils.py file rather to keep the views clean?

Agreed, I will move it.